### DISCUSSION TITLE
Guidance on Using CogVLM in a Python Script

### DISCUSSION THREAD

**Discussion by Kakachia777** [THUDM/CogVLM#232] [General]
I'm interested in integrating CogVLM into a Python script for a project and would appreciate some guidance. Could you provide some steps or documentation on how to set up and run CogVLM within a Python environment? Additionally, are there any specific considerations when making API calls to the model from Python?

Thank you for your support!

**Comment by zRzRzRzRzRzRzR** [comment]
Yes we have but it is a chinese doc in here https://zhipu-ai.feishu.cn/wiki/LXQIwqo1OiIVTykMh9Lc3w1Fn7g
